src/Microsoft.AspNetCore.OData/SingleResultOfT.cs

```diff
@@ -1,4 +1,11 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+//-----------------------------------------------------------------------------
+// <copyright file="SingleResultOfT.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Seems this file had a copyright header with .NET Foundation. You may need to remove this line and the one that follows manually